test/form-builder/components/ScriptEditorModal.spec.js

```diff
@@ -14,6 +15,8 @@ describe('ScriptEditorModal', () => {
   let updateScriptSpy;
   let closeSpy;
   let codeMirrorStub;
+  let beautifierStub;
+  let formatStub;
 
   beforeEach(() => {
```

Comment:
can we move the restore to aftereach, so that we make sure the cleanup happens even if no test is run afterwards